query/src/catalogs/table_meta.rs

```diff
@@ -21,7 +21,9 @@ use common_metatypes::MetaVersion;
 use crate::catalogs::Table;
 use crate::catalogs::TableFunction;
 
+//#[deprecated]
```

Comment:
What's this for?

Reply:
oh, forgot to rm this, I was planning to merge `TableMeta` into `Table`, since it provides no extra information. 
the plan has been postponed since issue #1949 do not care about it.